polyfill/lib/zoneddatetime.mjs

```diff
@@ -146,7 +146,7 @@ export class ZonedDateTime {
     const todayNs = GetSlot(ES.GetTemporalInstantFor(timeZone, today, 'compatible'), EPOCHNANOSECONDS);
     const tomorrowNs = GetSlot(ES.GetTemporalInstantFor(timeZone, tomorrow, 'compatible'), EPOCHNANOSECONDS);
     const { quotient, remainder } = tomorrowNs.subtract(todayNs).divmod(3.6e12);
-    return quotient.toJSNumber() + remainder.toJSNumber() / 3.6e12;
+    return (quotient.toJSNumber() * 3.6e12 + remainder.toJSNumber()) / 3.6e12;
```

Comment:
Maybe `quotient.multiply(3.6e12).add(remainder).toJSNumber() / 3.6e12` could be better, to keep everything in BigInt domain until the last division?

Reply:
Or, even easier, AFACT there's no need for the `divmod`, right?
```javascript
return tomorrowNs.subtract(todayNs).toJSNumber() / 3.6e12;
```

Code is updated with this solution.

Reply:
Oh right, that works, tomorrowNs − todayNs is guaranteed to be a safe integer.